### PR TITLE
libs/log: text logging format changes

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

**Review by alexanderbez — APPROVED** [review]

File: libs/log/default.go [hunk 1]
```diff
@@ -29,7 +30,17 @@ func NewDefaultLogger(format, level string, trace bool) (Logger, error) {
...
-		logWriter = zerolog.ConsoleWriter{Out: os.Stderr}
+		logWriter = zerolog.ConsoleWriter{
+			Out:        os.Stderr,
+			NoColor:    true,
+			TimeFormat: time.RFC3339,
+			FormatLevel: func(i interface{}) string {
+				if ll, ok := i.(string); ok {
+					return strings.ToUpper(ll)
+				}
+				return "????"
```
Comment (alexanderbez) [inline libs/log/default.go:41]: Wait, why are we doing this?

**Review by alexanderbez — CHANGES_REQUESTED** [review]

File: libs/log/default.go (hunk 1 above)
Comment (tychoish) [reply libs/log/default.go:41]: This is same semantics as the default behavior in zero log (use question marks for unknown/invalid levels).

Comment (tychoish) [reply libs/log/default.go:41]: see here: https://github.com/rs/zerolog/blob/98f889fcded426be57acc4aab62d615a2d679f0b/console.go#L331

**Review by marbar3778 — COMMENTED** [review]
change log, and please add a description to PRs. It would avoid questions as to why something is happening..

**Comment by tychoish** [comment]
I'm going to add a change log entry to the improvements section, noting that  the logging format changes. Having said that, the console log format _already_ changed as a result of #6534 and while the change log calls out the Go API, the formatting impact isn't called out (and I think that's fine.)

**Review by alexanderbez — APPROVED** [review]

**Review by marbar3778 — APPROVED** [review]

**Pull request merged**